templates/cluster-template.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: cluster.x-k8s.io/v1beta1
 kind: Cluster
```

Comment:
@Prajyot-Parab, would it make sense to move constructing the `cluster-template-vpc.yaml` to the e2e folder(as done for PowerVS) and only have the default template for VPC(`cluster-template.yaml`) here to avoid code duplication since using LB is the default and the templates are the same now?

Reply:
Noted, working on it.

Reply:
@Amulyam24 PTAL